app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -7,38 +7,30 @@
 define(function (require, exports, module) {
   'use strict';
 
-  const BaseView = require('views/base');
   const Notifier = require('lib/channels/notifier');
   const Session = require('lib/session');
+  const Url = require('lib/url');
 
   var Mixin = {
     notifications: {
       // populated below using event name aliases
     },
 
     clearSessionAndNavigateToSignIn () {
-      // Unset uid otherwise it will henceforth be impossible
-      // to sign in to different accounts inside this tab.
-      this.relier.unset('uid');
-      this.relier.unset('email');
```

Comment:
> If that's the case, are we sure we don't want to propagate the email here? It seems like it might prevent some frustration for these users, if we think the majority who sign back in made a mistake.

Ignore this, I misunderstood how the sign-in form behaves. I thought propagating the email was required for it to be set on the sign-in view, but that's working even after I took this out.